src/ext-pmu.adoc

```diff
@@ -602,6 +602,73 @@ The possible error codes returned in `sbiret.error` are shown in
                             <<_shared_memory_physical_address_range_parameter>>.
 |===
 
+=== Function: Get PMU Event info (FID #8)
+
+[source, C]
+----
+struct sbiret sbi_pmu_event_get_info(unsigned long shmem_phys_lo,
+                                     unsigned long shmem_phys_hi,
+                                     unsigned long num_entries,
+                                     unsigned long flags)
+----
+
+Get details about any PMU event via shared memory. The supervisor software can
+get event specific information for multiple events at one shot by writing an
+entry for each event in the shared memory. Each entry in the shared memory
+must be encoded as follows:
+
+[#table_event_info_format]
+.Event info entry format
+[cols="1,2,7", width=90%, align="center", options="header"]
+|===
+| Word | Name        | Encoding
+| 0    | event_idx   | BIT[0-19] - Describes `event_idx`
```

Comment:
Suggest changing "BIT[0-19]" to "BIT[0:19]", using the same bit format representation as other sections.
The same applies to the reset of BIT[] below.

Reply:
Done. PTAL.